experimental/packages/opentelemetry-api-events/src/types/EventEmitter.ts

```diff
@@ -0,0 +1,8 @@
+import { Attributes } from '@opentelemetry/api';
+import { Event } from './Event';
+
+export interface EventEmitter {
+  createEvent(name: string): Event;
+  emit(event: Event): void;
+  emit(name: string, attributes: Attributes): void;
```

Comment:
1. This `emit` method here should be called `logEvent` as per the API spec. 
2. There cannot be a method that takes `Event` as input, as `Events` can only be created from this API interface.

Reply:
> This emit method here should be called logEvent as per the API spec.

I have not had a chance to update this yet based on the latest version of the OTEP.

> There cannot be a method that takes Event as input, as Events can only be created from this API interface.

That's why I added the `createEvent` method to satisfy the requirement that events can be created only through this API. This is similar to the Tracer API in JS, which has `startSpan` method. I don't think we want to use the builder pattern in JS, so this is what I came up with, but I am open to other ideas.